app/src/main/java/fr/free/nrw/commons/theme/BaseActivity.java

```diff
@@ -27,6 +28,9 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         wasPreviouslyDarkTheme = systemThemeUtils.isDeviceInNightMode();
         setTheme(wasPreviouslyDarkTheme ? R.style.DarkAppTheme : R.style.LightAppTheme);
+        float fontScale = android.provider.Settings.System.getFloat(getBaseContext().getContentResolver()
+                , android.provider.Settings.System.FONT_SCALE, 1f);
```

Comment:
The convention is to not start a line with `,`. You can change to this for instance:
```
        float fontScale = android.provider.Settings.System.getFloat(
            getBaseContext().getContentResolver(),
            android.provider.Settings.System.FONT_SCALE,
            1f);
```